migrations/20190107123190-create-profile.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Profiles', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    firstName: {
+      type: Sequelize.STRING
+    },
+    lastName: {
+      type: Sequelize.STRING
+    },
+    bio: {
+      type: Sequelize.TEXT
+    },
+    imageUrl: {
+      type: Sequelize.STRING
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      onDelete: 'CASCADE',
+      references: {
+        model: 'users',
```

Comment:
Out of curiosity, I haven't created my migrations this way before, but I want to be sure that `users` will reference the correct `Users` model when need be.

Does the case matter?

Reply:
I need more clarification on question

Reply:
ok... yes it will reference the correct Users

Reply:
Do you have a `users` model? In the user model file, what I can see you defined is `User`

Reply:
it is referencing the migration file of the users which is named 'users'

Reply:
From my end, here, I can see the migration file named `20190107123130-create-user.js`

Unless I am missing something.

Reply:
I tested with  models name User, while runing migration it says relationship "User" doesn't exist, but when I used the migrations name or table name "users", migrations works fine...

Reply:
Okay then.